cartridges/int_affirm_pipelines/cartridge/scripts/payment/instrument/BASIC_CREDIT.js

```diff
@@ -26,7 +26,7 @@ exports.add = function (basket) {
     var cardNumber = parametersMap.number.stringValue;
     var cardHolder = parametersMap.cardholder_name.stringValue;
     var cardSecurityCode = parametersMap.cvv.stringValue;
-    var cardType = 'Master Card';
+    var cardType = 'Master';
```

Comment:
Same comment as above, I think this should be Visa

Reply:
Same as [above comment](https://github.com/Affirm/Salesforce_Commerce_Cloud_Affirm/pull/25#discussion_r427668858)